client/rest-high-level/src/main/java/org/elasticsearch/client/eql/EqlSearchRequest.java

```diff
@@ -121,6 +132,14 @@ public String implicitJoinKeyField() {
         return this.implicitJoinKeyField;
     }
 
+    public Boolean isCaseSensitive() { return this.isCaseSensitive; }
```

Comment:
Can you keep the same code style as in the other methods?
```
    public Boolean isCaseSensitive() {
        return this.isCaseSensitive;
    }
```

Reply:
0e742deb1a7b2c09701debfc17e7dc8f7423ad3e